### DISCUSSION TITLE
JS Override in App Settings

### DISCUSSION THREAD

**Discussion by mehmetuken** [directus/directus#3596] [Feature Requests]
I want adding custom js app because web push integration. Any suggestions?
Onesignal vs.

**Comment by benhaynes** [comment]
We added a Custom CSS option to the Project Settings — I know @rijkvanzanten was thinking about also adding a Custom JavaScript option too. So hopefully this comes soon!

**Reply by mehmetuken** [reply]
Thanks for fast answer. I'm waiting this future thanks.

**Reply by philleepflorence** [reply]
Just noticed this also yesterday, it would definitely make the transition from D8 to D9 smooth!
Might play around with the app to see how easy it would be to add this...

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
